eli5/sklearn/explain_weights.py

```diff
@@ -412,3 +414,13 @@ def _features(target_id):
             method='linear model',
             is_regression=True,
         )
+
+
+@explain_weights.register(Pipeline)
+def explain_weights_pipeline(estimator, feature_names=None, **kwargs):
+    last_estimator = estimator.steps[-1][1]
+    transform_pipeline = Pipeline(estimator.steps[:-1])
+    feature_names = transform_feature_names(transform_pipeline, feature_names)
```

Comment:
The passed feature names (sort of) takes priority here and is merely transformed, except where there is a vectoriser in the pipeline. But I think I should be using eli5's get_feature_names here to allow a vectoriser here. Is that correct?